core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/BPMBillingProjectDeleteRunner.scala

```diff
@@ -92,16 +92,17 @@ class BPMBillingProjectDeleteRunner(
   ): Future[JobStatus] =
     Try(workspaceManagerDAO.getDeleteLandingZoneResult(jobId.toString, lzId, ctx)) match {
       case Failure(e: ApiException) =>
-        val (msg, jobStatus) = e.getCode match {
-          case 500 => (s"Landing Zone deletion operation with jobId $jobId failed: ${e.getMessage}", Incomplete)
-          case 404 => (s"Unable to find jobId $jobId in WSM for Landing Zone deletion", Complete)
+        e.getCode match {
+          case 500 =>
+            val msg = s"Landing Zone deletion operation with jobId $jobId failed: ${e.getMessage}"
```

Comment:
The 500 case is now basically the same as the one below at line 103. I'm not sure if there is value in handling the 500 separately. I think that including the jobId and the status code in the message is probably sufficient.

Reply:
Makes sense to me
In the first pass, I was using the generic jobs endpoint, where all of this gets hit a lot more frequently, because the generic job endpoint returns its error reports as non-2xx
Since this is using the job-specific endpoint, 'normal' errors are returned with 200 and an error report, so there's a lot less value in handling specific error cases now